### PR TITLE
Enhancement Process Status

### PR DESCRIPTION
**Pull request by scelton**
Save documents into the database regardless the results of the process,
but 2 new columns have been added so we can store the status SUCCESS or
FAILURE, and status_reason containing the Exception in case of FAILURE.

### CODE REVIEW

**Comment by scelton** [comment]
Hi @Layoric, when you get a chance could u pls have a quick look through.

File: src/main/scripts/create-schema.sql [hunk 1]
```diff
@@ -2,10 +2,15 @@ CREATE SCHEMA IF NOT EXISTS hydroid AUTHORIZATION hydroid;
 
 CREATE TABLE hydroid.documents (
```
Comment (Layoric) [inline src/main/scripts/create-schema.sql]: `content bytea NOT NULL` might change given storage of originals in S3, but can be incorporated later.

**Comment by Layoric** [comment]
Looks good to me :+1:

File: src/main/scripts/create-schema.sql (hunk 1 above)
Comment (scelton) [inline src/main/scripts/create-schema.sql]: Agree, will remove.

**Pull request merged**